src/api/tests/queryMocks.ts

```diff
@@ -48,13 +48,26 @@ export const mockUser = (user: PartialUserOrNull | PartialUserOrNull[] = {}) =>
               id: 1,
               email: 'test_user@example.com',
               name: 'Test User',
-              participantId: 1,
+              participants: u.participants ?? [
```

Comment:
These mocks are still used by approversMiddleware and businessContactService tests. This change will allow those tests to still work, since it relies on logic in `hasBusinessContactAccess` instead.